### PR TITLE
Fix/14449 14455 SRS Consent Screen

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes to SRS consent screen 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14449
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14455

## Screenshots
Screenshot 2022-12-15 at 15 05 53

### CODE REVIEW

**Review by puneetmahali1 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 1]
```diff
@@ -512,12 +512,13 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		)
 		
 		let footerViewModel = FooterViewModel(
-			primaryButtonName: AppStrings.ExposureSubmissionTestResultAvailable.primaryButtonTitle,
-			primaryIdentifier: AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton,
-			isSecondaryButtonEnabled: false,
-			isSecondaryButtonHidden: true
+			primaryButtonName: AppStrings.SRSConsentScreen.primaryButtonTitle,
+			secondaryButtonName: AppStrings.SRSConsentScreen.secondaryButtonTitle,
+			primaryIdentifier: AccessibilityIdentifiers.SRSConsentScreen.primaryButton,
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:517]: Please attend to UI Test `test_SRSFlow_PositiveWithoutResultInTheApp()`: Identifier:

```swift
app.buttons[AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton].waitAndTap(.long)

// to 
app.buttons[AccessibilityIdentifiers.SRSConsentScreen.primaryButton].waitAndTap(.long)
```

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift [hunk 2]
```diff
@@ -512,12 +512,13 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		)
 		
 		let footerViewModel = FooterViewModel(
-			primaryButtonName: AppStrings.ExposureSubmissionTestResultAvailable.primaryButtonTitle,
-			primaryIdentifier: AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton,
-			isSecondaryButtonEnabled: false,
-			isSecondaryButtonHidden: true
+			primaryButtonName: AppStrings.SRSConsentScreen.primaryButtonTitle,
+			secondaryButtonName: AppStrings.SRSConsentScreen.secondaryButtonTitle,
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:516]: <3

**Review by flxschmidt — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift:517]: Fixed

**Review by 30mar — APPROVED** [review]

**Review by flxschmidt — APPROVED** [review]
👍

**Pull request merged**